nextflow_schema.json

```diff
@@ -156,6 +156,17 @@
                     "fa_icon": "far fa-file-code",
                     "description": "Path to fusioncatcher references"
                 },
+                "fusioncatcher_build": {
+                    "type": "boolean",
+                    "description": "Build fusioncatcher references instead of downloading them. This is required for any references other than for human gencode v46 at the moment."
+                },
+                "fusioncatcher_download_link": {
+                    "type": "string",
+                    "description": "Fusioncatcher download link",
+                    "fa_icon": "far fa-file-code",
+                    "pattern": "^https?://.*$",
+                    "default": "https://master.dl.sourceforge.net/project/rnafusion/fusioncatcher"
+                },
```

Comment:
We should eventually have our own references hosted in nf-core. https://github.com/nf-core/rnafusion/issues/515

Reply:
Yep that's the end goal!! Let's hope soon